cosmoz-omnitable-column-time.html

```diff
@@ -142,7 +142,8 @@ <h3 style="margin: 0;">[[ title ]]</h3>
 				if (value == null) {
 					return;
 				}
-				value = this.toValue(this._fixedDate + 'T' + value);
+				value = this.getAbsoluteISOString(this._fixedDate + 'T' + value);
```

Comment:
no need to put on two lines?
```suggestion
				value = this.toValue(this.getAbsoluteISOString(this._fixedDate + 'T' + value));
```